cmd/katalyst-agent/app/agent.go

```diff
@@ -159,7 +167,7 @@ func acquireLock(genericCtx *agent.GenericContext, conf *config.Configuration) *
 	for {
 		lock, err := general.GetUniqueLock(conf.LockFileName)
 		if err != nil {
-			_ = genericCtx.EmitterPool.GetDefaultMetricsEmitter().StoreInt64(metricsNameLockingFailed, 1, metrics.MetricTypeNameRaw)
+			_ = genericCtx.EmitterPool.GetDefaultMetricsEmitter().StoreInt64(metricsNameLockingFailed, 1, metrics.MetricTypeNameCount)
```

Comment:
why use count type instead of raw type here?

Reply:
because the raw type always report the last value, so for a certain instance it always report the value 1 no matter how many times it restarts. Count type can reflect the frequency. In this scenario, we should care about the restart frequency.

Reply:
should we add agent-component as tag here?

Reply:
do you mean the  agent-component like sysadvisor or qrm-plugins in bytedance environment?

Reply:
@csfldf sorry. I just noticed I changed this line by mistake, I thought this is the new metric. I'll revert this change.

Reply:
> do you mean the agent-component like sysadvisor or qrm-plugins in bytedance environment?

yeah, otherwise how will you distinguish between different component?

Reply:
in bytedance environment, we have psm. For open source users, it seems like we have no concept of agent component for now. Users can combine any agents they want together?